Portfolio: Deskstop version buildup

V0pe opened the pull request:
**Portfolio: Deskstop  version buildup**

In this project, I performed the following tasks;

- Update the index/html to accommodate the new text and structure.
- Add the new CSS file for the desktop.
- Update the style CSS to fix the error on mobile layout.
- Fix the linters error

Commit:
Update the markup for deskstop

Commit:
Add the deskstop style

Commit:
Fix mobile style error

Commit:
Fix linters error

Commit:
Fix broken image

omar-muhamad reviewed (changes requested):
# Changes Required ♻️
### Hi @V0pe,

You have done a great job so far, you are almost there.

![](https://media.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif)

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please add the mobile styling and desktop styling inside one CSS file that's because you are now repeating a lot of code inside each CSS file and you suppose to make your code DRY as much as possible.
The best way to do it is to make your whole styling in one CSS file by adding mobile styling first then add media query like this `@media screen and (min-width: 1080px)` for desktop styling and please note that it doesn’t have to be exactly 1080px, any value between 768 and 1440 is ok.

- [ ] Please fix the right margin inside your mobile version design because it misaligns all page elements and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/159945792-bbb06097-8e01-4400-95c8-493b15bf100e.png)  | ![image](https://user-images.githubusercontent.com/68796232/159945827-9c8d61f6-8825-4c00-bf0e-019004cf60f7.png)  |

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

V0pe commented:
Hi @Omar-Muhamad Thanks for your honest review. I added the desktop CSS separately from the mobile layout. If I just use the media query for some elements that has the same layout, I will end up using one CSS sheet. I linked the two CSS sheets at the HTML head tag. 

For the second review, I have adjusted the margin. 

Thanks.

Attached is the fixed result.
![image](https://user-images.githubusercontent.com/61356487/159951741-6dfabded-c90f-4bc0-b91c-11d6e24558a9.png)

Commit:
Fix layout issue

gemmen29 commented on index.html:
```diff
@@ -5,25 +5,32 @@
-    <header class="header">
-        <a href="#">Victor</a>
-        <nav class="navbar">          
+    <header class="header-container">
```
Kindly fix the right spacing as the previous reviewer asked
![image](https://user-images.githubusercontent.com/24703171/159958931-a259c592-6254-4479-a55a-b5a77445ea3f.png)
The problem happens because of the header section, you can solve it by:
- [ ] Kindly remove the `margin-right: 20px` of the `hamburger` button button.
- [ ] Please use the `display: none` property for the `ul` that contains desktop links instead of `visibility: hidden` and remove `display: flex` in the mobile version.
- [ ] Please use the `display: flex` property for the `ul` that contains desktop links instead of `visibility: visible` in the desktop version.

V0pe replied:
@gemmen29  Thanks for your review. I believe your observation and review are the best practice using ### display: none.

Attached is the screenshoot of the revision accordingly.  
![image](https://user-images.githubusercontent.com/61356487/160079945-fd4c4736-cf45-48f5-80d6-ba399ccc2a70.png)


_

gemmen29 reviewed (changes requested):
Hi @V0pe, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

![](https://media.giphy.com/media/tKyrc0NTlEcGWdThp4/giphy.gif)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Fix the layout issue

andersonlebon reviewed (approved):
## Hi @V0pe 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

![nodding-yes](https://user-images.githubusercontent.com/65068771/143095390-344d1cb0-b853-4b91-8662-5d9049dc9e60.gif)

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
